Description: Use a button for donations instead of an image
 of the documentation. Commenting out the relevant <li> tag prevents lintian
 from raising several privacy-breach-donation errors.
Author: Paul Wise
Bug-Debian: https://bugs.debian.org/873432
Forwarded: not-needed
Last-Update: 2021-12-28

--- a/ft2docs/docs/design/design-1.html
+++ b/ft2docs/docs/design/design-1.html
@@ -163,8 +163,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/design/design-2.html
+++ b/ft2docs/docs/design/design-2.html
@@ -218,8 +218,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/design/design-3.html
+++ b/ft2docs/docs/design/design-3.html
@@ -370,8 +370,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/design/design-4.html
+++ b/ft2docs/docs/design/design-4.html
@@ -332,8 +332,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/design/design-5.html
+++ b/ft2docs/docs/design/design-5.html
@@ -474,8 +474,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/design/design-6.html
+++ b/ft2docs/docs/design/design-6.html
@@ -340,8 +340,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/design/index.html
+++ b/ft2docs/docs/design/index.html
@@ -165,8 +165,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/documentation.html
+++ b/ft2docs/docs/documentation.html
@@ -111,8 +111,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/ft2faq.html
+++ b/ft2docs/docs/ft2faq.html
@@ -621,8 +621,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/glyphs/glyphs-1.html
+++ b/ft2docs/docs/glyphs/glyphs-1.html
@@ -198,8 +198,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/glyphs/glyphs-2.html
+++ b/ft2docs/docs/glyphs/glyphs-2.html
@@ -407,8 +407,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/glyphs/glyphs-3.html
+++ b/ft2docs/docs/glyphs/glyphs-3.html
@@ -455,8 +455,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/glyphs/glyphs-4.html
+++ b/ft2docs/docs/glyphs/glyphs-4.html
@@ -210,8 +210,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/glyphs/glyphs-5.html
+++ b/ft2docs/docs/glyphs/glyphs-5.html
@@ -378,8 +378,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/glyphs/glyphs-6.html
+++ b/ft2docs/docs/glyphs/glyphs-6.html
@@ -462,8 +462,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/glyphs/glyphs-7.html
+++ b/ft2docs/docs/glyphs/glyphs-7.html
@@ -360,8 +360,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/glyphs/index.html
+++ b/ft2docs/docs/glyphs/index.html
@@ -215,8 +215,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/index.html
+++ b/ft2docs/docs/index.html
@@ -267,8 +267,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/tutorial/index.html
+++ b/ft2docs/docs/tutorial/index.html
@@ -153,8 +153,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/tutorial/step1.html
+++ b/ft2docs/docs/tutorial/step1.html
@@ -1029,8 +1029,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/tutorial/step2.html
+++ b/ft2docs/docs/tutorial/step2.html
@@ -1545,8 +1545,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
--- a/ft2docs/docs/tutorial/step3.html
+++ b/ft2docs/docs/tutorial/step3.html
@@ -124,8 +124,8 @@
         <input type="hidden"
                name="hosted_button_id"
                value="SK827YKEALMT4">
-        <input type="image"
-               src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif"
+        <input type="submit"
+               value="Donate"
                name="submit"
                alt="PayPal - The safer, easier way to pay online!">
       </form>
